src/main/kotlin/com/openlattice/graph/Graph.kt

```diff
@@ -936,7 +915,7 @@ private val DELETE_BY_VERTICES_SQL = "$DELETE_SQL $VERTICES_FILTER_SQL"
 private val CLEAR_BY_VERTICES_SQL = "$CLEAR_SQL $VERTICES_FILTER_SQL"
 private val LOCK_BY_VERTICES_SQL = "$LOCK_SQL1 $VERTICES_FILTER_SQL $LOCK_SQL2"
 
-private val ASSOCIATIONS_FILTER_SQL = "${EDGE_ENTITY_SET_ID.name} = ? AND ${ID_VALUE.name} = ANY(?) AND ${COMPONENT_TYPES.name} = ${ComponentType.EDGE.ordinal} "
+private val ASSOCIATIONS_FILTER_SQL = "${ID_VALUE.name} = ANY(?) AND ${EDGE_ENTITY_SET_ID.name} = ? AND ${COMPONENT_TYPES.name} = ${ComponentType.EDGE.ordinal} "
 
 private val CLEAR_BY_ASSOCIATIONS_SQL = "$CLEAR_SQL $ASSOCIATIONS_FILTER_SQL"
```

Comment:
Have the functions calling `CLEAR_BY_ASSOCIATIONS_SQL` / `DELETE_BY_ASSOCIATIONS_SQL` / `LOCK_BY_ASSOCIATIONS_SQL` been updated for the reordering of id, entity set id ?